### ISSUE TITLE
[BUG] All LEGO games - res. reverts back to 640x368

### ISSUE THREAD

**Issue by Electry** [Electry/VitaGrafixPatchlist#1]
If you set IB res. to anything other than 480x272, 640x368, 720x408 or 960x544, it will default back to 640x368.

This is a known issue with all LEGO games that support IB res. change.
It is fixable, but different aspect ratios are going to stretch the image even after patching the res. check.
So after the fix, 840x476 would work, but 960x368, for example, would result in a stretched image.
Furthermore, different resolutions mess up the touchscreen, so that would also have to be fixed.

Unless there is a great interest in running the game at res. that isn't one of the above, I am not gonna fix this.

**Comment by nad45** [comment]
Hi @Electry
Lego Hobbit IB patch doesn't work
I added the "960*544" resolution in the config.txt, but when i launch the game, Vitagrafix says that the resolution is "960*5441", so as you said the resolution reverts back to 640*368.

With LEGO Batman, vitagrafix says that the resolution is 960*544 but i'm not sure that the resolution has really changed. I can't see any difference
Same sor Lego Avengers

**Comment by Electry** [comment]
you could try 480x272, that should be immediately obvious (if it works)

**Comment by nad45** [comment]
I tried with 480x472 and 960x544, i can see the difference, thanks.
But the resolution change is not as obvious than with Lego Star Wars, the game looks a lot better at native resolution.

And regarding the Hobbit do you know why Vitagrafix add a 1 at 944?

**Comment by Electry** [comment]
I don't know, I just tried it with EU version of The Hobbit and it worked fine for me. Does it also add "1" with other resolutions? If so, can you send me log.txt right after running the game? Thank you

**Comment by nad45** [comment]
I tried with 94, and it doesn't add 1
With 944,  a 1 is added, only with Hobbit EU version.
Regarding Lego Batman, i can see the difference between 480x272 and «960x544» but i can't see any difference between original resolution and native resolution, could you try?
I'll give you the Hobbit log tomorow, i have to go...
Thanks for your help. :)

**Comment by Electry** [comment]
You are absolutely right. Just did a side by side comparison and batman 3 renders at 640x368 even if you set it to 960x544. Looking at the star wars game, as it has the same engine, I forgot to add a part of the patch. Also opening up the character menu will freeze the vita at the moment.

Affected are: Jurassic World, Batman 3, Ninjago and Marvel's Avengers.

Fix coming tomorrow. Thanks for letting me know, I was trying to add a few games at once without testing each properly.

**Comment by ChiSys** [comment]
Could you add Urban Trial Freestyle aswell?

**Comment by nad45** [comment]
Hi @electry
I rebooted my Vita, so I don't have 960x5441 resolution with Lego Hobbit EU version.
But like the other LEGO games, i can't see any diference.
And the game randomly crashes.

**Comment by Electry** [comment]
did you re-download the new patchlist.txt?

**Comment by nad45** [comment]
No,i didn't know there is a new file.
I'm going to try
Thanks

**Comment by nad45** [comment]
I have re-downloaded the new patchlist.txt, it's working
Thank you
I have tried Lego Batman and Lego Avengers and i can really see the difference now!
Regarding Lego Hobbit, i can't see any difference, the resolution doesn't change while vitagrafix says it's 960x544
